src/checkbox-sk/checkbox-sk.ts

```diff
@@ -51,16 +51,20 @@ import { define } from '../define';
 import { upgradeProperty } from '../upgradeProperty';
```

Comment:
Per discussion on the other PR, the type annotations can be removed from all JSDoc.

Reply:
Done.